internal/jptrace/translator.go

```diff
@@ -15,10 +16,22 @@ import (
 func ProtoFromTraces(traces ptrace.Traces) []*model.Batch {
 	batches := jaegerTranslator.ProtoFromTraces(traces)
 	spanMap := createSpanMapFromBatches(batches)
-	transferWarnings(traces, spanMap)
+	transferWarningsToModelSpans(traces, spanMap)
 	return batches
 }
 
+// ProtoToTraces converts Jaeger model batches ([]*model.Batch)
+// to OpenTelemetry traces (ptrace.Traces).
+func ProtoToTraces(batches []*model.Batch) (ptrace.Traces, error) {
```

Comment:
do we actually want these functions to return an error? It seems like upstream ptrace functions never do, so this could be a convenient way to get rid of the error for the rest of Jaeger code.

Reply:
@yurishkuro great suggestion - removed